package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "developer-license-donotus",
+  "version": "1.0.0",
+  "description": "Lorem ipsum...",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
update to run forge test

Reply:
forge tests get run in a different way. this is only for the deploy script that's written in js